tests/ci/integration/run_ntp_integration.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -exu
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+source tests/ci/common_posix_setup.sh
+
+# Set up environment.
+
+# SYS_ROOT
+#  - SRC_ROOT(aws-lc)
+#    - SCRATCH_FOLDER
+#      - ntp
+#      - AWS_LC_BUILD_FOLDER
+#      - AWS_LC_INSTALL_FOLDER
+#      - NTP_BUILD_FOLDER
+
+# Assumes script is executed from the root of aws-lc directory
+SCRATCH_FOLDER="${SRC_ROOT}/NTP_BUILD_ROOT"
+NTP_VERSION="ntp-4.2.8p17"
```

Comment:
perhaps this is too brittle, but you could try to auto-detect the latest version and fall back to this one if there's no match

```
$ set -o pipefail
$ curl -s https://www.ntp.org/downloads/ | grep -oP 'The current release of NTP is <a href=".+">([0-9a-z\.]+)' | cut -d'>' -f2
4.2.8p17
```

Reply:
Let's see how stable it is 👍  I went with a slightly different approach to get both the folder and specific version.